content-scripts/src/modules/schedule.js

```diff
@@ -0,0 +1,437 @@
+import { injectOverrideFunctions } from "./initialize";
+import { lcmAll } from "./utilities/math";
+import removeElement from "./utilities/removeElement";
```

Comment:
```suggestion
import { lcmAll } from "./utilities/math";
```

Remove unused imports